rotkehlchen/tasks/calendar.py

```diff
@@ -109,60 +113,91 @@ def get_history_events(self, event_types: list[tuple[HistoryEventType, HistoryEv
 
         return events
 
-    def maybe_create_calendar_entry(self, event: 'EvmEvent', name: str, description: str, timestamp: Timestamp, color: 'HexColorCode', counterparty: str) -> int | None:  # noqa: E501
-        """Create calendar entry from an event.
-        Returns the id of the entry created or None if no entry was created"""
-        assert event.location_label is not None
-        if (
-            (user_address := string_to_evm_address(event.location_label)) not in self.blockchain_accounts.get(  # noqa: E501
-                blockchain := ChainID.deserialize(event.location.to_chain_id()).to_blockchain(),
-            ) or timestamp <= self.current_ts
-        ):
-            return None  # Skip events in the past or from a different address
-
+    def maybe_create_calendar_entry(
```

Comment:
Change the name to reflect what's happening

```suggestion
    def create_or_update_calendar_entry(
```